src/StrawberryShake/Tooling/src/dotnet-graphql/InitCommandHandler.cs

```diff
@@ -134,5 +141,26 @@ await arguments.AuthArguments
                 configuration.ToString())
                 .ConfigureAwait(false);
         }
+
+        private static Dictionary<string, IEnumerable<string>> ParseHeadersArgument(List<string?> arguments)
```

Comment:
```suggestion
        private static Dictionary<string, IEnumerable<string>> ParseHeadersArgument(
            List<string?> arguments)
```

Reply:
Moved to a helper class since I added support for custom headers in download and update commands.

Reply:
Fixed